screens/AllTasks.tsx

```diff
@@ -19,16 +18,10 @@ export default function AllTasks() {
         "SELECT category FROM Tasks GROUP BY category"
       );
       setCategs(allCategs);
-
-      // for(const cat of allCategs){
-      //   const categTasks =await db.getAllAsync<Task>("SELECT * FROM Tasks WHERE category = ? ",[cat.category]);
-
-      //   setCategs((curState)=>{return(     )    })
-      // }
     }
 
     fetchCats();
-  }, []);
+  }, [db]);
 
   console.log(categs);
```

Comment:
Remove logs, they should be limited to your development only. Never push debugging logs to remote.